testing/keep_largest_object.py

```diff
@@ -0,0 +1,101 @@
+import os
```

Comment:
A short description of the script could be added.

Reply:
done here ! https://github.com/ivadomed/model_seg_sci/pull/87/commits/bda85800d01e87240ffd7647d3be6cc4840b5e92